api/userauth.go

```diff
@@ -133,3 +133,20 @@ func (g GithubAuthHandler) GithubAuth(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	w.Write(userJSON)
 }
+
+func (c ClusterHandler) AltAuth(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "POST" {
+		http.Error(w, "Invalid Method ", http.StatusMethodNotAllowed)
+		return
+	}
+	apiKeyHeader := r.Header.Get("x-api-key")
+	log.Println("inside altauth")
```

Comment:
Looks good, we can remove this log line (I assume it's to help with debugging) since we're not really inside the function.